tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -122,7 +122,9 @@ def _run(self):
                     )
 
         while not self._complete.is_set():
-            gevent.sleep(10)
+            gevent.sleep(
```

Comment:
Currently there are no ttl references at the moment for entities in ceph-integration. When required we can use as `sync_interval + 15`